src/frontend/package.json

```diff
@@ -33,7 +33,7 @@
     "@typescript-eslint/parser": "^4.5.0",
     "babel-eslint": "^10.1.0",
     "classnames": "^2.2.6",
-    "czifui": "^1.2.1",
+    "czifui": "^1.4.0",
```

Comment:
did you consider pinning the latest version (1.5.0)?

Reply:
I am happy to pin the latest version, `1.4.0` was just the latest when I changed it.